debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSourceInfoStructMaker.java

```diff
@@ -45,7 +45,7 @@ public Struct struct(SourceInfo sourceInfo) {
             result.put(SourceInfo.TXID_KEY, sourceInfo.txId());
         }
         if (sourceInfo.lsn() != null) {
-            result.put(SourceInfo.LSN_KEY, sourceInfo.lsn());
+            result.put(SourceInfo.LSN_KEY, sourceInfo.lsn().asLong());
```

Comment:
Ok, I really hope we don't miss this somewhere.

Reply:
+1